src/test/java/org/terasology/blockdetector/systems/README.md

```diff
@@ -0,0 +1,19 @@
+Block Detector Tests
+===================
+
+This tests are developed to test some methods in the BlockDetectorSystemImpl Class. The methods that are tested are update(), initialise, addDetector and detectBlocks.
```

Comment:
`This tests` -> `These tests`

You've added () after update, but not after the other 3. Either add it after every method name, or remove it after update as well.